examples/framework-react/astro.config.mjs

```diff
@@ -4,5 +4,5 @@ import react from '@astrojs/react';
 // https://astro.build/config
 export default defineConfig({
 	// Enable React to support React JSX components.
-	integrations: [react()],
+	integrations: [react({experimentalReactChildren: true})],
```

Comment:
```suggestion
	integrations: [react()],
```
I'm guessing this was used for debugging. We wouldn't want to enable this for everyone creating a new project with the `framework-react` template.

Reply:
Same for the other changes in `/examples`, it's where we keep the starter templates seen on [astro.new](https://astro.new).

Reply:
Got it I wasn't sure what the rules were on that and I also thought it might be useful to see the changes in action. Should I revert all changes made within the examples folder?

Reply:
yes

Reply:
I've gone ahead and removed that, hope that's okay.